### PR TITLE
Add runbook doc for setting up a fork of the project

### PR DESCRIPTION
**Pull request by jivdhaliwal**
# What
Adds instructions on how to fork the repository and keep it synced with the upstream repo

# Why
To help people setup their own instance of the app

# Screenshots

# Notes

### CODE REVIEW

File: docs/runbook/setting-up-your-repo.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+![fork button](../images/fork_button.png)
+
+Each trust has a fork of the madetech/nhs-virtual-visits repository, allowing your own feature development, and ability to receive updates from Made Tech.
+
+As time goes on, the trust fork will become out of date with the upstream as we deliver security patches, and possibly new features. You can either install a utility to keep your fork up to date automatically, or manually sync the fork yourself.
+
+## Automatically updating your fork
+
+We have set up the repository to work instantly with the [Pull app by Wei](https://github.com/wei/pull). To install the app, visit https://github.com/apps/pull and click install, pointing the app to your newly created fork.
```
Comment (antonyoneill) [inline docs/runbook/setting-up-your-repo.md:18]:
```suggestion
We have set up the repository to work instantly with the [Pull app by Wei](https://github.com/wei/pull). To install the app, visit [github.com/apps/pull](https://github.com/apps/pull) and click install, pointing the app to your newly created fork.
```

File: docs/runbook/setting-up-your-repo.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Manually updating your fork
+
+You can also manually sync your fork with the upstream repo via the following instructions: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork
```
Comment (antonyoneill) [inline docs/runbook/setting-up-your-repo.md:22]:
```suggestion
You can also manually sync your fork with the upstream repo via the following instructions: [Github: Syncing a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork)
```

File: docs/runbook/setting-up-your-repo.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Caveats
+
+If you make changes to your forked repo, the Pull utility (or manual sync) will attempt to merge our changes into yours. Any conflicting changes will need to be resolved (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/addressing-merge-conflicts).
```
Comment (antonyoneill) [inline docs/runbook/setting-up-your-repo.md:26]:
```suggestion
If you make changes to your forked repo, the Pull utility (or manual sync) will attempt to merge our changes into yours. Any conflicting changes will need to be resolved (see [Github: Addressing merge conflicts](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/addressing-merge-conflicts)).
```

File: docs/runbook/setting-up-your-repo.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Setting up your repository
+
+To run your own version of the NHS Book a virtual visit app, you will need to first create a fork of the repository
```
Comment (antonyoneill) [inline docs/runbook/setting-up-your-repo.md:3]: Not sure if we need any guidance on creating a github account and an organisation.. May be too much detail

Comment (jivdhaliwal) [reply docs/runbook/setting-up-your-repo.md:3]: If you try and fork while logged out you'll be taken to the login / signup page so I think we can leave that out but we could link to https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams to show how to setup an org

Comment (antonyoneill) [reply docs/runbook/setting-up-your-repo.md:3]: That's good enough in my opinion.

**Pull request merged**